### PR TITLE
New Quickstart: Ray

### PR DESCRIPTION
**Pull request by pkudikyala**
**Jira Ticket:** https://new-relic.atlassian.net/browse/NR-249390
**Description:** Created Ray new QuickStart.

# Summary

A concise description of the changes being introduced. Please review the pre-merge checklist section to validate this pull request is ready for review and merge. **If it is not ready, please mark the pull request as a draft.**

_The owners of this repo are not experts in the subject matter of the quickstarts. We review for the quickstart to be functional and for security risks. If you are seeking feedback on the content of the quickstart, please seek out a subject matter expert. If you are not an internal NR contributor, we can do our best to connect you with a content reviewer._

## Pre merge checklist

- [ ] Did you check you NRQL syntax? - Does it work?
- [ ] Did you include a Data source and Documentation reference?
- [ ] Are all documentation links publicly accessible?
- [ ] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [ ] Did you check your descriptive content for spelling and grammar errors?
- [ ] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [ ] Does the PR contain a screenshot for each of your dashboards?
- [ ] Do your screenshots show data?
- [ ] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [ ] Did you check that your alerts actually work?

### CODE REVIEW

File: alert-policies/ray/ActiveNodes.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Name of the alert
+name: Ray Active Nodes
+
+# Description and details
+description: |+
+  This alert is triggered if the active nodes lesser than 1 for 5 minutes.
```
Comment (seemantk) [inline alert-policies/ray/ActiveNodes.yml:6]: Unclear what is the meaning of this

File: alert-policies/ray/FreeDiskPercentage.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# Name of the alert
+name: Ray Free Disk Percentage
+
+# Description and details
+description: |+
+  This alert is triggered if the free disk percentage lesser than 10% for 5 minutes.
```
Comment (seemantk) [inline alert-policies/ray/FreeDiskPercentage.yml:6]:
```suggestion
  This alert is triggered if there is less than 10% of free disk space for 5 minutes.
```

File: data-sources/ray/config.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+id: ray
+displayName: Ray
+description: Identify and optimize Ray performance with New Relic Ray monitoring.
```
Comment (seemantk) [inline data-sources/ray/config.yml:3]:
```suggestion
description: Monitor the performance of Ray using New Relic monitoring.
```

**Review by seemantk — CHANGES_REQUESTED** [review]

File: quickstarts/ray/config.yml [hunk 4]
```diff
@@ -0,0 +1,44 @@
+slug: ray
+description: |
+  ## Comprehensive monitoring quickstart for Ray
+  By integrating Ray with New Relic, you can get comprehensive information about the quality of your codebase and receive alerts about serious vulnerabilities. Enabling you to effectively monitor performance, promptly diagnose issues, and optimize your application for improved efficiency and scalability.
```
Comment (mdumpati) [inline quickstarts/ray/config.yml:4]:
```suggestion
  By integrating Ray with New Relic, you can get comprehensive information about the quality of your codebase and receive alerts about critical vulnerabilities. Enabling you to effectively monitor performance, promptly diagnose issues, and optimize your application for improved efficiency and scalability.
```

File: alert-policies/ray/ActiveNodes.yml (hunk 1 above)
Comment (pkudikyala) [reply alert-policies/ray/ActiveNodes.yml]: Hi @seemantk,
I have rephrased the description. Could you please review it?

File: alert-policies/ray/FreeDiskPercentage.yml (hunk 2 above)
Comment (pkudikyala) [reply alert-policies/ray/FreeDiskPercentage.yml]: Hi @seemantk ,
As per your suggestion, I have made the changes. Could you please review it?

File: data-sources/ray/config.yml (hunk 3 above)
Comment (pkudikyala) [reply data-sources/ray/config.yml]: Hi @seemantk ,
As per your suggestion, I have made the changes. Could you please review it?

File: quickstarts/ray/config.yml (hunk 4 above)
Comment (pkudikyala) [reply quickstarts/ray/config.yml]: Hi @mdumpati  ,
As per your suggestion, I have made the changes. Could you please review it?

**Review by mdumpati — APPROVED** [review]
LGTM

**Review by seemantk — APPROVED** [review]

**Pull request merged**